lib/Doctrine/ORM/Internal/Hydration/AbstractHydrator.php

```diff
@@ -151,9 +152,8 @@ public function iterate($stmt, $resultSetMapping, array $hints = [])
     /**
      * Initiates a row-by-row hydration.
      *
-     * @return iterable<mixed>
-     *
      * @psalm-param array<string, mixed> $hints
+     * @psalm-return Generator<int, (array|mixed)>
```

Comment:
:thinking: the union of `array` and `mixed` is `mixed`, so what's the point?

Reply:
fixed